CMakeLists.txt

```diff
@@ -62,8 +62,8 @@ endif()
 option(FMILIB_BUILD_STATIC_LIB "Build the library as static." ON)
 option(FMILIB_BUILD_SHARED_LIB "Build the library as shared (dll/so/dylib)." ON)
 option(FMILIB_INSTALL_SUBLIBS  "Install sub-libraries (fmicapi. fmixml, jmutil, fmiimport, fmizip)" OFF)
-option(FMILIB_USE_NATIVE_ZLIB  "Use native installation of zlib, possibly requires setting ZLIB_LIBRARY & ZLIB_INCLUDE_DIR to locate installation." OFF)
-mark_as_advanced(FMILIB_USE_NATIVE_ZLIB)
+option(FMILIB_FIND_PACKAGE_ZLIB  "Use find_package for zlib, instead of version in FMILibrary/Thirdparty, possibly requires setting ZLIB_LIBRARY & ZLIB_INCLUDE_DIR." OFF)
```

Comment:
I think it's better to skip the last part with `, possible requires setting ...`. If things stop working when using this option, then a user can read the documentation on ` find_package` and `FindZLIB.cmake` (the errors about missing ZLIB_LIBRARY originates from there).

Reply:
Fixed.